packages/components-react/color-sample-react/src/color-sample.tsx

```diff
@@ -0,0 +1,23 @@
+import { clsx } from 'clsx';
+import type { HTMLAttributes } from 'react';
+import { forwardRef } from 'react';
+
+export interface ColorSampleProps extends HTMLAttributes<HTMLDataElement> {
+  value: string;
+}
+
+export const ColorSample = forwardRef<HTMLDataElement, ColorSampleProps>(function ColorSample(props, forwardedRef) {
+  const { value, children, className, style, ...restProps } = props;
```

Comment:
@Robbert We zouden `value` hier (en op L17) weg kunnen laten om dat `restProps` 'm toch wel spread. Thoughts?

Reply:
Prima!